src/email/interfaces/email.service.interface.ts

```diff
@@ -0,0 +1,5 @@
+import { Email } from "../dto/email.dto";
+
+export interface IEmailService {
```

Comment:
Since we have already abstract EmailService class, we don't need this interface. We can remove it.